_health-care/_js/store/calculated.js

```diff
@@ -0,0 +1,7 @@
+// Functions that produce calculated values given a state.
+//
+
+export function neverMarried(state) {
+  return state.nameAndGeneralInformation.maritalStatus === '' ||
```

Comment:
In Joe's documentation of the original form, apparently `maritalStatus` is a required field, so maybe we don't need this calculation?